docs/rules/no-extra-parens.md

```diff
@@ -23,6 +23,7 @@ This rule has an object option for exceptions to the `"all"` option:
 * `"conditionalAssign": false` allows extra parentheses around assignments in conditional test expressions
 * `"returnAssign": false` allows extra parentheses around assignments in `return` statements
 * `"nestedBinaryExpressions": false` allows extra parentheses in nested binary expressions
+* `"ignoreJSX": "none|all|multiline|single-line"` allows extra parentheses around all/multiline/single-line JSX components. Defaults to `none`.
```

Comment:
I think the option name should be `multi-line` for consistency with `single-line`.